src/Microsoft.OData.Core/Evaluation/ODataResourceMetadataContext.cs

```diff
@@ -135,14 +135,15 @@ public IODataResourceTypeContext TypeContext
             ODataResourceSerializationInfo serializationInfo,
             IEdmStructuredType actualResourceType,
             IODataMetadataContext metadataContext,
-            SelectedPropertiesNode selectedProperties)
+            SelectedPropertiesNode selectedProperties,
+            ODataMetadataSelector metadataSelector = null)

```

Comment:
>ODataMetadataSelector metadataSelector = null [](start = 12, length = 45)

(minor) Since this is internal (and only called in two places outside of tests) I would generally rather explicitly pass null than specify a default.